parameter_sync.py

```diff
@@ -167,4 +187,43 @@ def get_parameters_with_prefix(prefix, next_token=None):
     logger.info('Parameter values will be compared against file contents in "%s" and updated if necessary' % args.cred_path)
     process_parameters_with_prefix(args.param_prefix, args.cred_path, args.aws_region,
                                    args.aws_access_key, args.aws_secret_key, args.dryrun)
+
+    if args.save_aws_creds:
+        if args.aws_access_key and args.aws_secret_key:
+            aws_access_key = args.aws_access_key
+            aws_secret_key = args.aws_secret_key
+        else:
+            aws_access_key = os.environ.get('AWS_ACCESS_KEY_ID')
+            aws_secret_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
+
+        if len(args.save_aws_creds) % 2 != 0:
+            logging.critical('Must provide both filename and profile name for aws creds')
+            sys.exit(1)
+
+        aws_creds_tuples = []
+        filename = None
+        profile = None
+        for i, val in enumerate(args.save_aws_creds):
+            if (i % 2) == 0:
```

Comment:
The idea here is that if you use the --save-aws-creds option, you need to provide two inputs, the filename to save to and the profile name. The first input is the filename, the second in the profile name